src/v/cloud_storage/remote_partition.h

```diff
@@ -77,9 +77,14 @@ class remote_partition
     /// offsets. All offset translation is done internally. The returned record
     /// batch reader will produce batches with kafka offsets and the config will
     /// be updated using kafka offsets.
+    /// \param config is a reader config
+    /// \param deadline is an optional time point which is used to debounce
+    ///        readers which are waiting for an offset. This is not a
+    ///        cancellation deadline.  Not used for tiered-storage.
```

Comment:
is this true? it looks to me like remote_partition::make_reader ignores the deadline...?